### PR TITLE
python-decouple added.

### PR DESCRIPTION
**Pull request by xmnlab**

### CODE REVIEW

File: recipes/python-decouple/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+test:
+  imports:
+    - decouple
+
+about:
+  home: https://github.com/henriquebastos/python-decouple/
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (dougalsutherland) [inline recipes/python-decouple/meta.yaml]: Could you send a PR upstream adding this to the distribution and put a comment with its URL here? Just need to add a `MANIFEST.in` file with the line `include LICENSE`, and possibly a `setup.cfg`, like https://github.com/torchbox/django-libsass/pull/28.

**Comment by dougalsutherland** [comment]
Looks good after my comment above.

File: recipes/python-decouple/meta.yaml (hunk 1 above)
Comment (xmnlab) [reply recipes/python-decouple/meta.yaml]: done! thank you @dougalsutherland !

File: recipes/python-decouple/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - mock
```
Comment (synapticarbors) [inline recipes/python-decouple/meta.yaml]: It seems like `mock`, `pytest`, `tox`, `docutils` and `pygments` are all testing requirements and aren't actually needed to run this package. If this is the case, please remove these from the `run` section.

Comment (xmnlab) [reply recipes/python-decouple/meta.yaml]: @synapticarbors thank you for the advice! :+1:

**Comment by xmnlab** [comment]
thanks @isuruf

**Pull request merged**